frontend/src/components/pages/GlobalCodeReviewDashboard/FilterByNames.jsx

```diff
@@ -13,7 +13,7 @@ export default function FilterByNames({
 
   return (
     <React.Fragment>
-      {allNames.map((name,index) => (
```

Comment:
Please add the textbox here. 

If we are using the FilterByNames component 3 times then there should be three textboxes.